playbook-website/app/javascript/site_styles/_site-style.scss

```diff
@@ -164,27 +164,26 @@ body {
   }
 
   &--footer {
-    background-color: $royal;
-    padding: $space_lg * 2 $space_lg * 3;
+    background-color: $bg_dark;
+    padding: $space_lg * 2 0;
     margin-top: auto;
     .inner-footer-wrapper {
-      max-width: 1350px;
       margin: 0 auto;
     }
+    .link-container {
+      width: 100%;
+    }
```

Comment:
Is this fine to have until we create the width prop?

Reply:
Yeah. I think it's fine for now, and we will adjust it once I get the `width` global prop sorted out.